src/github.js

```diff
@@ -144,22 +146,28 @@ export default class GitHub {
 				"Contributors, please [read how to link your accounts](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/) to ensure your work is properly credited in WordPress releases.\n\n";
 		}
 
-		commentMessage += "## Core SVN\n\n" +
-		"If you're a Core Committer, use this list when committing to `wordpress-develop` in SVN:\n" +
-		"```\n" +
-		"Props: " + contributorsList['svn'].join(', ') + "." +
-		"\n```\n\n" +
-		"## GitHub Merge commits\n\n" +
-		"If you're merging code through a pull request on GitHub, copy and paste the following into the bottom of the merge commit message.\n\n" +
-		"```\n";
+		if ( this.format == 'svn' || this.format == 'both' ) {
```

Comment:
Thinking about forward compatibility, would it make sense for `format` to be an array and thus to check if `svn` is in that array or maybe see if that array contains `all`? This allows for additional formats beyond "github" and "svn"

Reply:
My plan was to default to `github`, but also accept `svn` and `both`.

I think an array could also work. Though I'm not sure what other formats we'd introduce in the near future.

Reply:
Maybe one day we will finally switch to mercurial :) 

My thinking with making it an array rather than a string check is less about the near future and more about the long future.